packages/Meshfree/test/tstNearestNeighborExodusGenerator.cpp

```diff
@@ -204,7 +204,9 @@ void testUniquelyOwnedProblem(
 
     // Create a nearest neighbor operator.
     DataTransferKit::NearestNeighborOperator<Device> nearest_op(
-        comm, src_coords_device, tgt_coords_device );
+        *( Teuchos::rcp_dynamic_cast<Teuchos::MpiComm<int> const>( comm )
+               ->getRawMpiComm() ),
```

Comment:
Do we use `Teuchos::MpiComm` in this test?

Reply:
The test use free functions from `Teuchos_CommHelpers.hpp` and builds a `Tpetra::Map`